index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <br/>
+        <button class="see-project">See Project</button>
+      </div> <br>
+    <section class="about-me">
+      <h2 class="about-me-title">About me</h2>
+      <p class="about-me-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </section> <br>   
+    <div class="card-2">
```

Comment:
- [ ] Just like you have lists for the Languages list, Frameworks list, and Skills list, each of these 3 parts should also be list items of one list container. The 3 of the `div` elements with `class="card-2"` should be replaced with `li` elements and all of them wrapped in a `ul` element. Kindly make that change.